autosklearn/estimators.py

```diff
@@ -253,11 +253,11 @@ def fit(self, *args, **kwargs):
         self._automl = self.build_automl()
         super(AutoSklearnEstimator, self).fit(*args, **kwargs)
 
-    def fit_ensemble(self, task=None, metric=None, precision='32',
+    def fit_ensemble(self, y, task=None, metric=None, precision='32',
```

Comment:
I see your point. After thinking about this for a little bit, I think it's more important to have the interface for regression and classification as similar as possible. Even if this implies that we have to pass an unused `y` to `fit_ensemble`. Do you see any reason why this is bad?